### PR TITLE
Json attributes setter

### PR DESCRIPTION
**Pull request by wswoodruff**
Add a setter that sets `_jsonAttributesMemo` under the hood.

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -249,3 +249,110 @@ internals.uniqueObjects = (arr) => {
 
     return uniqueObjects;
 };
+
+exports.Model = class SchwiftyModel extends Objection.Model {
```
Comment (devinivy) [inline lib/index.js]: No need to re-add the entire model into lib/index.js.  You can go ahead and make the changes in lib/model.js.

**Review by devinivy — CHANGES_REQUESTED** [review]
The build here is quite broken, https://travis-ci.org/BigRoomStudios/schwifty/jobs/192882873#L252

Looks like a bad merge.  Probably cause I moved the model on you!  Check lib/model.js.

File: lib/index.js (hunk 1 above)
Comment (wswoodruff) [reply lib/index.js]: Let's see...

Comment (wswoodruff) [reply lib/index.js]: rofl whoops

**Comment by wswoodruff** [comment]
Once a PR steals a commit hash, you can't change the history, you gotta contact Github staff. So gotta switch some stuff real quick:

**Comment by wswoodruff** [comment]
Hm, shoulda checked tests before pushing

**Comment by devinivy** [comment]
Fixes #18

**Review by devinivy — APPROVED** [review]

**Comment by wswoodruff** [comment]
Sweet 👍

**Pull request merged**